lib/offline_geocoder.rb

```diff
@@ -0,0 +1,46 @@
+require "offline_geocoder/version"
+require "csv"
+require "kdtree"
+
+class OfflineGeocoder
+  def initialize
+    unless defined? @@cities
```

Comment:
It will be always in memory, certainly. This gem in oriented to mass geocoding, so this makes sense.